base/intset.jl

```diff
@@ -239,3 +245,8 @@ function hash(s::IntSet, h::UInt)
     end
     h
 end
+
+minimum(s::IntSet) = first(s)
+maximum(s::IntSet) = last(s)
+extrema(s::IntSet) = (first(s), last(s))
+issorted(s::IntSet) = true
```

Comment:
For `IntSet`, the iteration order is guaranteed to stay fixed; if the implementation changes, then the `issorted` method will have to be updated accordingly.